### DISCUSSION TITLE
Increase Font Size and Bold Font

### DISCUSSION THREAD

**Discussion by shockergit** [iSoron/uhabits#648] [Feature Requests]
Can you please add a font changing option in the app like small, medium large and extra large this will help people will less visibility.
Also can you give option to make the font bold. please.

**Comment by JackElsey** [comment]
I can understand why the font has to be small in the visualizations, but it is unnecessarily tiny in the measurable habit update dialog.

**Comment by herrbrixter** [comment]
I want a bigger font because you can hardly see the colors and can't distinguish which one is pass or fail. The text is too small.
